src/api/vdom.js

```diff
@@ -44,17 +44,22 @@ function applyEvent (elem, ename, newFunc) {
     events = elem[$currentEventHandlers] = {};
   }
 
-  const oldFunc = events[ename];
-
-  // Remove old listener so they don't double up.
-  if (oldFunc) {
-    elem.removeEventListener(ename, oldFunc);
+  // Undefined indicates that there is no listener yet.
+  if (typeof events[ename] === 'undefined') {
```

Comment:
It's definitely not a breaking change. The API stays the same; nothing changes for the user.